### PR TITLE
feat: add Telemetry implementation for Sample App

### PR DESCRIPTION
**Pull request by guitarrapc**
## TL;DR

Adding Open Telemetry(MagicOnion) + Prometheus + Grafana implementation.
Also adding cAdvisor to collect container metrics. Optionally adding node_exporter to collect host metrics for Linux.

Also as previous sample add Unity 2018.3.6f1 had affedted critical bug due to synchronization context, I've update to latest, 2019.1.10f1, and confirm it fixed.

## Summary

Now sample implements OpenTelemetry-net w/Prometheus and  metrics can retieve on http://localhost/9182.

This PR also prepared docker-compose to offer you Prometheus to collect metrics and Grafana to visually show metrics on dashboard.

## TODO

- [x] Add implementations
- [x] Add Prometheus to collect metrics
- [x] Add Grafana to visualize metrics
- [x] Add cAdvisor to collect container metrics
- [x] optionarlly add node_exporter to collect host metrics.
- [x] Add README for Telemetry
- [x] Add user defined default tags support.
- [x] Remove Sample's OpenTelemetry.cs.

### CODE REVIEW

File: samples/ChatApp/ChatApp.Server/ChatHub.cs [hunk 1]
```diff
@@ -33,20 +33,35 @@ public async Task LeaveAsync()
...
 
             await Task.CompletedTask;
         }
 
+        public Task GenerateException(string message)
+        {
+            throw new Exception(message);
+        }
+
+        public Task UnaryGenerateException(string message)
```
Comment (vivilily) [inline samples/ChatApp/ChatApp.Server/ChatHub.cs]: Please delete it because it is not used.

File: samples/ChatApp/ChatApp.Unity/Assets/Scripts/ServerShared/Hubs/IChatHub.cs [hunk 2]
```diff
@@ -16,6 +17,10 @@ public interface IChatHub : IStreamingHub<IChatHub, IChatHubReceiver>
 
         Task SendMessageAsync(string message);
 
+        Task GenerateException(string message);
+
+        Task UnaryGenerateException(string message);
```
Comment (vivilily) [inline samples/ChatApp/ChatApp.Unity/Assets/Scripts/ServerShared/Hubs/IChatHub.cs]: Please delete it because it is not used.

File: samples/ChatApp/ChatApp.Server/ChatHub.cs (hunk 1 above)
Comment (guitarrapc) [reply samples/ChatApp/ChatApp.Server/ChatHub.cs]: done.

File: samples/ChatApp/ChatApp.Unity/Assets/Scripts/ServerShared/Hubs/IChatHub.cs (hunk 2 above)
Comment (guitarrapc) [reply samples/ChatApp/ChatApp.Unity/Assets/Scripts/ServerShared/Hubs/IChatHub.cs]: done

**Comment by guitarrapc** [comment]
I've received request to merge this implements to `MagicOnion.OpenTelemetry/MagicOnionCollector.cs`.
Will do this.
